.gitignore

```diff
@@ -0,0 +1 @@
+media
```

Comment:
media directory should never be in the repository. contains files from our website users